### PR TITLE
Added phone-number exercise.

### PR DESCRIPTION
**Pull request by cpaulbond**
Let me know if there are any problems.

### CODE REVIEW

**Comment by zenspider** [comment]
This looks good to me @kytrinyx / @canweriotnow

File: exercises/phone-number/example.el [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+(defun prefix (num)
+  (substring (numbers num) 3 6))
+
+
+(defun line-number (num)
+  (substring (numbers num) 6))
+
+
+(defun pprint (num)
+  (concat "(" (area-code num) ") " (prefix num) "-" (line-number num)))
```
Comment (wobh) [inline exercises/phone-number/example.el]: If this were an Exercism submission, I'd suggest using `format` here.

**Comment by canweriotnow** [comment]
I'm leaving this open for the moment... I agree with @wobh that `format` would be a better exemplar... although this works, so I'm not against merging... If anyone has thoughts one way or the other, chime in please.

**Comment by yurrriq** [comment]
Thanks!

**Pull request merged**